Cargo.toml

```diff
@@ -17,11 +17,12 @@ derive_more     = "^0.14"
 hex             = "^0.3"
 log             = "^0.4"
 openssl         = { version = "^0.10", optional = true }
-quick-xml	= "^0.14"
+quick-xml	    = "^0.14"
 ring            = "^0.13"
-serde		= "^1.0"
+serde   		= "^1.0"
+serde_derive   	= "^1.0"
```

Comment:
The serde guide doesn’t include `serde_derive` separately but rather does this:

```
 serde = { version = "1.0", features = ["derive"] }
```

(and then `#[macro_use] extern crate serde` in lib.rs).

Reply:
pushed